pages/common/chmod.md

```diff
@@ -1,36 +1,24 @@
 # chmod
 
-> Change the access permissions of a file or directory.
+> Change file/directory access permissions.
 > More information: <https://www.gnu.org/software/coreutils/chmod>.
 
-- Give the [u]ser who owns a file the right to e[x]ecute it:
+- Add [r]ead/[w]rite/e[x]ecute permissions for a file/directory owner:
 
-`chmod u+x {{file}}`
+`chmod +{{r|w|x}} {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
```

Comment:
This is a dangerous example that's equivalent to assigning 0777 permissions to something. This is not something we want to encourage at all.

Reply:
It is not clear for me what to consider dangerous and what not. I can consider all commands requiring `sudo` as dangerous ones but `chmod` is not a case by this logic.

Reply:
Right. In this case, I mean specifically that `0777` permissions are widely considered bad practice, as it allows *anyone* to write to and change a file. By including it as an example in a tldr page, we encourage it. Resources on this issue:

 - https://pimylifeup.com/chmod-777/
 - https://www.anchor.com.au/blog/2012/09/the-chmod-777-trap-how-and-why-to-avoid-it/